ethclient_utils/client.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ethclient_utils
```

Comment:
First, Go convention is to name packages in all lowercase with no punctuation, so `ethclientutils`.
Second, I'm not a huge fan of `ethclientutils`, since this package wraps `ethclient.Client`, rather than providing utilities on `ethclient.Client` instances. Honestly I think `ethclient` on its own is differentiated enough, since it is how other `awm-relayer` packages will create (`subnet-evm`) `ethclient.Clients`.

Reply:
I will make the change 👍 
In some places we still need to import `subnet-evm/ethclient` for type assertion. So I hope there will ne be confusion caused by package name collision between `subnet-evm` and `awm-relayer`.

Reply:
We can remove the dependency on `subnet-evm/ethclient` everywhere except for `ethclient/client.go` by wrapping the the `subnet-evm` client like so:
```
package ethclient
import "github.com/ava-labs/subnet-evm/ethclient"

type Client interface {
    ethclient.Client
}
```

Then everywhere in the rest of the repo that uses `subnet-evm/ethclient` can be replaced with `awm-relayer/ethclient`.

Reply:
I agree 👍